authors/apps/authentication/views.py

```diff
@@ -26,7 +32,26 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
 
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        # email = serializer.data.get('email', None)
+        email = serializer.data['email']
+        username = serializer.data['username']
+        token=serializer.data['token']
+         
+        sender = "ahfulldeck@gmail.com"
+        current_site = get_current_site(request)
+        verification_link = "http://" + current_site.domain + '/api/v1/verify/{}'.format(token)
+        subject = "Verification email"
+        message = "Hello {}, Click the following link to verify your account {}".format(username, verification_link)  
+
+        
+        send_mail(subject, message, sender, [email])
+
+        response_message = {
+            "message": "User registered successfully. Check your mail for verification",
+            "user_serializer": serializer.data
```

Comment:
@eduhmik change "user_serializer" here to "user info"

Reply:
Thanks for the feedback @kzyGit. 
I have made changes to settings.py file so that we can export email settings from the .env
I am working on the rest of the feedback

Reply:
Hey @kzyGit,  implemented feedback. Kindly review

Reply:
Implemented all feedback